src/main/resources/changelog/scripts/v-1.0.0/resource_graph/tables/resources.sql

```diff
@@ -11,6 +11,7 @@ CREATE TABLE if not exists resources (
   active_event_id bigint null,
   inventory_id uuid,
   srs_id uuid,
+  source resource_source  null,
```

Comment:
We will clean-up the DB and deploy again. Hence, no need to give an upgrade script.